Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java

```diff
@@ -43,6 +44,9 @@ public ShooterWheelSubsystem(
                 propManager.createPersistentProperty(side + " flush to boiler target speed", 9000);
         trimFlushToBoilerSpeed =
                 propManager.createEphemeralProperty(side + " trim speed", 0.0);
+        minWheelSpeed = 
+                propManager.createPersistentProperty("Minimum wheel speed", 9001);
```

Comment:
Explain what the minimum speed does; as is, it might seem like it defines the minimum speed at which the wheel can spin.